services/src/contexts/postgres.rs

```diff
@@ -2435,4 +2441,298 @@ mod tests {
         })
         .await;
     }
+
+    #[tokio::test(flavor = "multi_thread", worker_threads = 1)]
+    #[allow(clippy::too_many_lines)]
+    async fn test_types_individually() {
```

Comment:
what exactly do you test here? That you can insert it as a parameter and get it from a Row? Maybe you can write that in the the test name

Reply:
[0d9f680](https://github.com/geo-engine/geoengine/pull/844/commits/0d9f680b8e88caac754d9c099106bb8a7f2fd9aa)